docs/configuration.md

```diff
@@ -1519,6 +1519,17 @@ Apart from these, the following properties are also available, and may be useful
     you can set larger value.
   </td>
 </tr>
+<tr>
+  <td><code>spark.maxRemoteBlockSizeFetchToMem</code></td>
```

Comment:
There is already an entry in this doc for this we just need to update the default/description, see above

Reply:
There seems to be just one entry for this at the moment